tests/v2/validation/snapshot/README.md

```diff
@@ -16,35 +16,32 @@ rancher:
   clusterName: "cluster_to_run_tests_on"
   insecure: true/optional
   cleanup: false/optional
+snapshotInput:
+  upgradeKubernetesVersion: "v1.27.6+rke2r1" # If left blank, the default version in Rancher will be used.
+  snapshotRestore: "all" # Options include none, kubernetesVersion, all.
```

Comment:
does this have a default like upgradeKubernetesVersion? I think the comment should specify that 'none' means etcd only. Otherwise, 'none' could sound like it would skip the restore

Reply:
I can add that comment to clarify